circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/Replica.java

```diff
@@ -87,6 +89,7 @@
     tableFactory = replicaTableFactory;
     this.housekeepingListener = housekeepingListener;
     this.replicationMode = replicationMode;
+    environmentContext = new EnvironmentContext(); // TODO if we have to pass properties, which ones should we pass?
```

Comment:
@ddcprg what is the choice here?

Reply:
there is nothing we can do here for now

Reply:
OK, so we leave the TODO here for later?

Reply:
I think, as a reminder

Reply:
OK, fine by me.